From 4d42f7b414f9cf15051509a80fd8ecdd745b989c Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Mon, 20 Dec 2021 18:46:33 +0100
Subject: [PATCH] Fixup Imath 3.0 support

OpenEXR_FOUND needs to be set to true in order to compile it.
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4a77c4be50b..89636205c2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -341,6 +341,7 @@ endif ()
 macro_optional_find_package(Imath 3.0 CONFIG QUIET)
 if(TARGET Imath::Imath)
     set(OPENEXR_LIBRARIES Imath::Imath)
+    set(OpenEXR_FOUND TRUE)
 else()
     macro_optional_find_package(OpenEXR)
 endif()
-- 
GitLab

